docs/self-hosted/oel/oauth2/revert-database-migrations.mdx

```diff
@@ -0,0 +1,91 @@
+---
+id: revert-database-migrations
+title: Reverting database migrations
+---
+
+# Reverting Database Migration for Ory Hydra
+
```

Comment:
@christiannwamba PTAL as well

Reply:
Thanks. I will add to the OEL Nav PR (#2047)  when this is merged